docs/sources/panels/field-options/standard-field-options.md

```diff
@@ -86,6 +87,20 @@ You can also paste a native emoji in the unit picker and pick it as a custom uni
 
 Grafana can sometime be too aggressive in parsing strings and displaying them as numbers. To make Grafana show the original string create a field override and add a unit property with the `string` unit.
 
+## Color
+
+> Only available in Grafana 7.3+
+
+The field color option defines how Grafana should color series or fields. There are multiple modes here that work very differently and their utility depend largly on what visualization you currenly have selected.
+
+Continuous color modes use a value's percentage relative to min and max to interpolate a color.
+
+- **Fixed color**: Set a specific color using the color picker. Mostly useful from an override rule.
```

Comment:
- The colons belong in the bold tag with the text.
- Apply the serial comma in the last two bullets.
- Change the final field to sentence case.
- All visualization names need to be lowercase in this usage, because they are generic rather than proper names.

Reply:
:+1: